src/context_manager/context_manager.cpp

```diff
@@ -88,7 +95,14 @@ bool load_tokens(std::string &at, std::string &rt) {
             raw_result, TOKEN_CACHE_FILE_MAXSIZE);
   LogDebug("read size " << num_read << "\n");
 
-  if (!token_pair_from_string(raw_result, num_read, at, rt)) return false;
+  try {
+    if (!token_pair_from_string(raw_result, num_read, at, rt)) return false;
+  } catch (Autolab::CryptoException &e) {
+    Logger::fatal << "OpenSSL error in load_tokens." << Logger::endl;
```

Comment:
This is no longer a fatal error so not the most accurate, though I do see that there is only info / debug options for logging otherwise right now

Reply:
Switched to `LogDebug`

**Debug mode**
<img width="487" alt="Screenshot 2024-05-21 at 13 32 31" src="https://github.com/autolab/Autolab-CLI/assets/9074856/2aaecec2-1b93-4150-9e4f-85602b114ef3">

**Release mode**
<img width="460" alt="Screenshot 2024-05-21 at 13 33 48" src="https://github.com/autolab/Autolab-CLI/assets/9074856/9ec65eb0-80f1-408f-93d5-0c6ce0a0877f">